logger/logger.go

```diff
@@ -54,8 +54,11 @@ func New(appname string, opts Options) (Handler, error) {
 
 	// log.SetReportCaller(true)
 	log.SetOutput(os.Stdout)
-	log.SetLevel(logrus.InfoLevel)
+	if opts.Output != nil {
+		log.SetOutput(opts.Output)
+	}
 
+	log.SetLevel(logrus.InfoLevel)
 	if opts.DebugLevel {
 		log.SetLevel(logrus.DebugLevel)
```

Comment:
Do you have an example of how / where Debug will be flagged as "on"/"true"?

Reply:
In mesheryctl we need Debug to be turned on, as many times we print additional info with the flag `--verbose`